### PR TITLE
fix lumi profile

### PR DESCRIPTION
**Pull request by SimeonEhrig**
KelvinHelmholtz example compiles, but I don't know if it is running because I have no idea to launch the example.

### CODE REVIEW

**Comment by BrianMarre** [comment]
you should be able to run after compiling with the following:

```bash
tbg -s sbatch -c etc/picongpu/1.cfg -t etc/picongpu/lumi-eurohpc/standard-g.tpl ~/test_khi
```

**Comment by BrianMarre** [comment]
our tpl file for lumi also contains a comment that says it is intended for frontier, we might want to change that ...

**Comment by SimeonEhrig** [comment]
Okay, I found the typo in the `tgb` command. Now, I'm confirmed, that something is wrong with the MPI module:

```bash
tbg/submit.start: line 46: mpiexec: command not found
```

**Comment by PrometheusPi** [comment]
@SimeonEhrig strange - they must have changed something since last summer - thanks for fixing

**Comment by SimeonEhrig** [comment]
@PrometheusPi I think so.

image

File: etc/picongpu/lumi-eurohpc/lumi-G_hipcc_picongpu.profile.example [hunk 1]
```diff
@@ -48,13 +48,17 @@ module load buildtools/23.03
...
+export CC=clang
+export CXX=clang++
+
 export CXXFLAGS="$CXXFLAGS -I${MPICH_DIR}/include"
+export HIPFLAGS=$CXXFLAGS
 export LDFLAGS="$LDFLAGS -L${MPICH_DIR}/lib -lmpi ${PE_MPICH_GTL_DIR_amd_gfx90a} ${PE_MPICH_GTL_LIBS_amd_gfx90a}"
 
 # Other Software ##############################################################
 #
-libpng/1.6.38-cpeCray-23.03
```
Comment (PrometheusPi) [inline etc/picongpu/lumi-eurohpc/lumi-G_hipcc_picongpu.profile.example:57]: sorry - I never use png output and thus did not see this error

**Review by PrometheusPi — APPROVED** [review]

**Review by psychocoderHPC — APPROVED** [review]

**Pull request merged**